### PR TITLE
Releasing 0.47-stable branch

### PR DESCRIPTION
**Pull request by sayar**
Will also push the branch itself once the changes below have been reviewed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,22 +27,22 @@
   ],
   "peerDependencies": {
     "react": "16.0.0-alpha.12",
-    "react-native": "~0.46.0-rc.2"
+    "react-native": "~0.47.0-rc.4"
   },
   "devDependencies": {
     "flow-bin": "^0.47.0",
```
Comment (rozele) [inline package.json]: react-native@0.47.* uses flow-bin@0.49.1

Comment (rozele) [inline package.json]: You'll also need to update .flowconfig

**Comment by rozele** [comment]
@sayar you'll need to push a copy of the rntester branch to this repo so AppVeyor can reference it, can you make a separate pull request for that first?

**Comment by sayar** [comment]
Made all the changes you asked for and rebased into a clean history.

**Comment by rozele** [comment]
@sayar looks like the unit tests are not compiling, can you look into that as well?

**Comment by sayar** [comment]
@rozele woops, forgot to reinclude a file in the .projitems after renaming. Its fixed and rebased now.

**Comment by sayar** [comment]
Updated this to latest master branch (as of 7/27 3pm).

**Pull request merged**